Userland/Applications/BrowserSettings/AutoplaySettingsWidget.h

```diff
@@ -29,12 +32,20 @@ class AutoplaySettingsWidget : public GUI::SettingsWindow::Tab {
     virtual void apply_settings() override;
     virtual void reset_default_values() override;
 
+    // Generated from GML, don't call manually
+    static ErrorOr<NonnullRefPtr<AutoplaySettingsWidget>> try_create();
+    AutoplaySettingsWidget() = default;
+
 private:
-    explicit AutoplaySettingsWidget(NonnullRefPtr<AutoplayAllowlistModel>);
+    void set_allowlist_model(NonnullRefPtr<AutoplayAllowlistModel> model);
 
     RefPtr<GUI::Menu> m_entry_context_menu;
     RefPtr<GUI::CheckBox> m_allow_autoplay_on_all_websites_checkbox;
     RefPtr<GUI::Button> m_add_website_button;
     RefPtr<GUI::ListView> m_allowlist_view;
-    NonnullRefPtr<AutoplayAllowlistModel> m_allowlist_model;
+    RefPtr<AutoplayAllowlistModel> m_allowlist_model;
 };
+
+}
+
+using BrowserSettings::AutoplaySettingsWidget;
```

Comment:
If this is too big to convert to the namespace at once, it should probably be done up front instead of exporting this into a namespace where it doesn't belong.

Reply:
> a namespace where it doesn't belong.

I'm pretty sure nobody else uses BrowserSettings code, so the top-level namespace of an application is fair game for this?

Reply:
If nothing uses it, we can either put it into the namespace as a whole or omit the namespace completely. But we should pick _one_ of them.